### DISCUSSION TITLE
GPU Plugin - Color Blindness Intensity setting

### DISCUSSION THREAD

**Discussion by iatethedinosaurs** [runelite/runelite#16196] [Ideas]
The ability to specify the strength (%) level of colorblind filters in the GPU plugin would be amazing. In my case, the default Deuteranope setting is too intense. 117 has a similar option and I've been using that at 50%, but I prefer the GPU plugin and would love to see if there would be consideration to adding that option to the GPU plugin.

**Comment by bedmndsn** [comment]
I did some poking around in Java and AI coding and got this working!
#19882

**Comment by bedmndsn** [comment]
As of today this update is live 😀
